client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
+    bit_v = (8 * struct.calcsize("P"))
+    if (bit_v == 32):
+        print("32 bit Python is not supported. Please install the 64 bit version.")
+        log.debug("32 bit Python is detected.")
```

Comment:
Doing both logging and printing is kind of redundant here. Since we definitely want to print this message, you can omit the log message.